src/libsyntax/attr.rs

```diff
@@ -299,16 +299,16 @@ pub fn find_crate_name(attrs: &[Attribute]) -> Option<InternedString> {
 }
 
 /// Find the value of #[export_name=*] attribute and check its validity.
-pub fn find_export_name_attr(diag: &Handler, attrs: &[Attribute]) -> Option<InternedString> {
+pub fn find_export_name_attr(diag: Option<&Handler>, attrs: &[Attribute]) -> Option<InternedString> {
```

Comment:
Needs to be done https://github.com/rust-lang/rust/pull/30830/files#diff-896d2c044f21cf78ca23be2b4b6a52a1R240 and https://github.com/rust-lang/rust/pull/30830/files#diff-ccf7f52a9abbf3308db4fa771bd5c0ecR264

Can you advise how to get one from those contexts?